aggregate.go

```diff
@@ -108,13 +108,18 @@ type AggregateConfigurer interface {
 	// handler, such as application state or message routing information.
 	//
 	// The application and the handlers within it MUST have distinct keys. The
-	// key MUST NOT be changed. The RECOMMENDED key format is RFC 4122 UUID,
-	// generated when the handler is first implemented.
-	//
-	// Both the name and the key MUST be non-empty UTF-8 strings consisting
-	// solely of printable Unicode characters, excluding whitespace. A printable
-	// character is any character from the Letter, Mark, Number, Punctuation or
-	// Symbol categories.
+	// key MUST NOT be changed. The RECOMMENDED key format is an RFC 4122 UUID
+	// represented as a hyphen-separated, lowercase hexadecimal string, such as
+	// "5195fe85-eb3f-4121-84b0-be72cbc5722f".
+	//
+	// Both identifiers MUST be non-empty UTF-8 strings consisting solely of
+	// printable Unicode characters, excluding whitespace. A printable character
```

Comment:
Is whitespace actually a printable character?

Reply:
Yeah, as weird it sounds, whitespace is in the set of characters called "printable". It's distinguished from things like control characters.

Reply:
Cool, just wanted to specify, thanks.